### PR TITLE
phpMyAdmin: update to 5.2.0, use mariadb, change maintainer

### PR DESCRIPTION
**Pull request by williamdes**
#### Testing the changes
- I tested the changes in this PR: NO

I am trying to understand how void linux works, until now I can not figure out how to start nginx or apache

### CODE REVIEW

File: srcpkgs/phpMyAdmin/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-version=5.0.4
+version=5.2.0
 revision=1
 wrksrc="phpMyAdmin-${version}-all-languages"
 conf_files="/etc/webapps/phpMyAdmin/config.inc.php"
-depends="php mysql"
+depends="php mariadb"
 short_desc="Web interface for MySQL and MariaDB"
-maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
+maintainer="William Desportes <williamdes@wdes.fr>"
```
Comment (classabbyamp) [inline srcpkgs/phpMyAdmin/template:9]: @Francesco149 can they adopt this from you?

**Comment by TinfoilSubmarine** [comment]
If you'd like to adopt, could you include my fixed update check from https://github.com/void-linux/void-packages/pull/38984.

I don't use phpMyAdmin, so I was also curious about which PHP version should be used. PHP 7.4 is EOL in November, and at that point the `php` package will become a package depending on `php8.1`, effectively upgrading everyone with PHP 7.4 installed to 8.1. I'm tracking the progress of catching any errors that might happen from this in https://github.com/void-linux/void-packages/pull/38536.

All that said, if `phpMyAdmin` needs PHP 8.0 and not PHP 8.1, it might be a good idea to change the `php` dependency to `php8.0` to avoid an error state.

**Comment by williamdes** [comment]
> If you'd like to adopt, could you include my fixed update check from https://github.com/void-linux/void-packages/pull/38984.

Sure, but I would prefer that you include my change on your PR
See: https://github.com/void-linux/void-packages/pull/38984/files#r960994211

> All that said, if `phpMyAdmin` needs PHP 8.0 and not PHP 8.1, it might be a good idea to change the `php` dependency to `php8.0` to avoid an error state.

phpMyAdmin needs any PHP version between 7.1 and 8.2 ;)

Does void linux need packages to switch from `php` to `php8.1` ?

**Comment by TinfoilSubmarine** [comment]
> phpMyAdmin needs any PHP version between 7.1 and 8.2 ;)
> 
> Does void linux need packages to switch from `php` to `php8.1` ?

No, the `php` package isn't going anywhere, so leaving it as is should be fine.

**Comment by williamdes** [comment]
Replaced by #38984
Thanks @TinfoilSubmarine

File: srcpkgs/phpMyAdmin/template (hunk 1 above)
Comment (Francesco149) [reply srcpkgs/phpMyAdmin/template:9]: ok to adopt, I am not maintaining my void packages these days

**Pull request closed without merging**